src/app/components/challenge/challenge.component.html

```diff
@@ -1,150 +1,153 @@
+                                                <a class="pointer"
+                                                    (click)="deleteChallenge()">
+                                                    <i class="fa fa-trash red-text"
+                                                        aria-hidden="true"></i>
+                                                </a>
+                                            </span>
+                                            <br>
+                                            <span style="line-height: 1.5"
+                                                *ngIf="challenge.creator.team_url != ''"><strong
+                                                    class="text-light-black fw-semibold
+                                                    fs-15">Organized by:</strong><a
+                                                    class="orange-text"
+                                                    href="{{challenge['creator']['team_url']}}"
+                                                    target="_blank">
+                                                    {{challenge['creator']['team_name']}}
+                                                </a>
+                                            </span>
+                                            <span style="line-height: 1.5"
+                                                *ngIf="challenge.creator.team_url == ''"><strong
+                                                    class="text-light-black fw-semibold
+                                                    fs-15">Organized by:</strong>
+                                                {{challenge['creator']['team_name']}}
+                                            </span>
 
-                                <br>
-                                <span style="line-height: 1.5"
-                                    *ngIf="isChallengeHost">
-                                    <span>
-                                        <a class="pointer"
-                                            (click)="togglePublishChallengeState()">
-                                            <strong class="text-light-black
-                                                fw-semibold fs-15">{{publishChallenge.state}}
-                                            </strong>
-                                            <i class="{{publishChallenge.icon}}"
-                                                aria-hidden="true"></i>
-                                        </a>
-                                    </span>
-                                </span>
-                                <br>
-                                <mat-checkbox color="primary" class="fw-semibold" *ngIf="isChallengeHost"
-                                    [checked]="!challenge['is_registration_open']"
-                                    (click)="stopParticipation($event)">Close Participation
-                                </mat-checkbox>
```

Comment:
Hey, why have you removed `stop participation` option?

Reply:
I don't think I did.